app/components/pixi.js

```diff
@@ -399,6 +398,40 @@ megaBeamsprout() {
     // Generate a base texture from the base graphic
     const baseTexture = this.app.renderer.generateTexture(p);
     let particles = genParticles(baseTexture);
+    return [particles, reset, update, drops];
+  }
+
+  @action
+  didInsert() {
+    let type = "WebGL";
+    if (!PIXI.utils.isWebGLSupported()) {
+      type = "canvas";
+    }
+
+    PIXI.utils.sayHello(type);
+
+    this.app = new PIXI.Application({
+      view: document.getElementById("pixi"),
+      autoResize: true,
+      resolution: devicePixelRatio,
+      backgroundAlpha: 0,
+    });
```

Comment:
everything seems good except for this part I'm a bit worried about this leaking a bunch of pixi.application instances since these don't get cleaned up.